uber/templates/budget/index.html

```diff
@@ -10,13 +10,13 @@
 </style>
 
 <div class="jumbotron">
-    <h1 class="text-center">(${{ total }} total)</h1>
+    <h1 class="text-center">(${{ '%0.2f' % total }} total)</h1>
```

Comment:
`amount_paid` and `amount_refunded` are floats now, so I added this text formatting in a ton of places. I haven't checked them all to make sure these are actually integers -- I'll need to do that later.